roles/falcon_configure/README.md

```diff
@@ -100,10 +105,22 @@ Ensure the following API scopes are enabled (***if applicable***) for this role:
 - When using API credentials `falcon_client_id` and `falcon_client_secret`
   - **Sensor Download** [read]
 
+## Falcon Sensor Service
+
+For Linux/macOS, the Falcon Sensor service is automatically started after the sensor is installed (*although it will be in a failed state*).
+
+> [!WARNING]
+> The service will be in a failed state until the CID is set and the sensor is restarted.
+
+This role will restart the Falcon Sensor service after the CID is set, along with any other configuration options.
+
 ## Dependencies
 
 - Privilege escalation (sudo) is required for this role to function properly.
-- Falcon Sensor must be installed
+- The Falcon Sensor must be installed
+
+> [!IMPORTANT]
+> The role automatically handles privilege escalation so you do not need to set `become: true` in your playbook.
```

Comment:
This appears to directly conflict with the first bullet above. I think you're saying, "you need to be able to escalate, but do not escalate the entire role."

Reply:
This statement translates to:
DO NOT SET BECOME for these roles!

Reply:
How about something like:

```
- The Falcon Sensor must be installed
- Privilege escalation (sudo) is required for this role to function properly. Please make sure that it is configured in your environment. When the role is run, it will automatically handle privilege escalation so avoid setting `become` in your playbooks to prevent issues when the playbook is run.
```